assets/vue/Doughnut.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+    <Doughnut :chart-data="chartData" :chart-options="chartOptions" />
+</template>
+
+<script lang="ts" setup>
+    import { Doughnut } from "vue-chartjs";
+    import { Chart as ChartJS, registerables } from "chart.js";
+    ChartJS.register(...registerables);
+
+    interface Genre {
```

Comment:
hier bin ich immer noch für genereller machen. (umbennen zu "Item" oder so)

Reply:
Das ist ja in der GenreDoughnut Komponente. Nicht die Doughnut kopmponente... die ist in Assets.

Reply:
Das soll eigentlich ein Genre modellieren. Da seh ich kein Sinn drin das ein generischen Namen zu geben

Reply:
Diese Datei (Doughnut.vue) ist eine genereller Component. Demnach sollte auch ein genereller Name für so ein Item/Element kommen. Das habe ich bei mir auch gemacht (siehe bild)
![image](https://user-images.githubusercontent.com/103991412/194775724-3ca46a28-f516-4499-ac62-581891df64d5.png)

Reply:
Oh das tut mir echt leid, ich habe den file Name anscheinend falsch gelesen😂. Ich dachte ich hätte das sogar double-checked...

Du hast hier vollkommen recht, danke.